web-ui/view/organisation.ml

```diff
@@ -0,0 +1,52 @@
+open Git_forge
+
+module Make (M : M_Git_forge) = struct
+  let profile_picture_url org =
+    (* FIXME [benmandrew]: How can we get the GitLab profile pictures? *)
```

Comment:
Gitlab serves avatars from gravatar, you can query them via the REST API
eg
```
curl -X GET https://gitlab.com/api/v4/users\?username\=tmcgilchrist | jq   
[
  {
    "id": 490393,
    "username": "tmcgilchrist",
    "name": "Tim McGilchrist",
    "state": "active",
    "avatar_url": "https://secure.gravatar.com/avatar/67afd2b4c98c9befd18c19f0ee9d94dc?s=80&d=identicon",
    "web_url": "https://gitlab.com/tmcgilchrist"
  }
]
```

Reply:
Ah cool thanks - paging @benmandrew :)